plume-models/src/mentions.rs

```diff
@@ -3,7 +3,7 @@ use diesel::{self, ExpressionMethods, QueryDsl, RunQueryDsl};
 
 use comments::Comment;
 use notifications::*;
-use plume_common::activity_pub::inbox::Notify;
+// use plume_common::activity_pub::inbox::Notify;
```

Comment:
?

Reply:
I guess I temporally "disabled" it to see what errors I got or something like that, and I forgot to completely remove it. :grimacing: (I should probably review my own code before pushing it)